dts/bindings/sensor/st,lsm6dsr-i2c.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023, Andrei Hutanu
+# SPDX-License-Identifier: Apache-2.0
+
+description: STMicroelectronics LSM6DSR 6-axis accelerometer and gyrometer
+
+compatible: "st,lsm6dsr"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  irq-gpios:
+    # This signal is active high when produced by the sensor
+    type: phandle-array
```

Comment:
Oh, ok, already there... Please create a common file included by i2c and spi (we need this support) and add the gyro/xl fs/odr properties at least.